xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -537,6 +538,13 @@ boolean isShutDown() {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Returns the {@link TlsContextManager} used in this XdsClient.
+   */
+  TlsContextManager getTlsContextManager() {
+    throw new UnsupportedOperationException();
```

Comment:
cc\ @voidzcy for `XdsClient` API change.

Reply:
LGTM from the surface level. Will need to look inside to see how the TlsContextManager gets passed around and its lifecycle.